### DISCUSSION TITLE
Something in Script that causes VPN's to "Not connect" to a Server?

### DISCUSSION THREAD

**Discussion by leeuniverse** [memstechtips/UnattendedWinstall#144] [Q&A]
**UPDATE....**
The below might still be an issue, because I think it caused some "System Time" issue on my "first" Fresh Install of my OS with this script when opening ProtonVPN.
I think I might have told my OS to "sync" this time, so I didn't have that issue when opening Proton this time.

Turns out this time was TinyWall didn't actually "fully" disable when I allowed Traffic.  I had Enabled Blocklists in TinyWall which I never did before, and those remained active, despite letting Traffic through, so those blocks were actually blocking ProtonVPN.

Anyway, current issue fixed....  I'll leave below just in case there IS some sort of "System Time" issue caused by this script, it not "updating" properly or something, maybe because a Service was disabled that shouldn't have been?

Thanks....

-------

So, was using ProtonVPN on my Win10 Pro prior to a Fresh Install with this script (LTSC version), and had never had issues.
But I do a Fresh Install with this script, same OS and basic Programs, do all the normal Exclusions, disabling stuff etc. to make sure it connects to a Server initially.
But can't connect....  Did every single "Troubleshooting" thing I could find on Google and their Site, and even atm with their Tech support, though they've told me nothing I haven't tried so far.

Anyway, of the standard stuff removed, not seeing anything that should "effect" a VPN.
However, I'm wondering....  **Is maybe one of the "Services" that's been set to "Manual" could be causing this?**
This script disables etc. a million of them.... so maybe Proton is needing one of them running to work right?

BTW, I've tried the "Hide.me" VPN and it's program DOES connect thus work.  So, it's something with ProtonVPN, and I don't know if any other VPN's?

Thoughts?  Thanks

BTW....  On one of my Fresh Installs, was testing things, I did try ProtonVPN, and at that time it was giving me an error that my "Windows System Time" Clock whatever it's called was not accurate, was not synced to the correct time or whatever.
But, Proton wasn't giving that error this time around, it just wouldn't connect to a server, but it WOULD give an error saying I need to "Enable" stuff for the program to work like maybe every 6 times i opened the program.

So, I noticed in the script it "Removes" Last Access Time Stamp...
`:: Don't Update Last Access Time Stamp - This Can Improve File System Performance`
` fsutil.exe behavior set disableLastAccess 1`

Also, when Searching the File, I noticed there are "5 other" Time related "Services" that are set to Manual.
` Set-Service -Name 'TimeBroker' -StartupType Manual -ErrorAction Continue`
` Set-Service -Name 'TimeBrokerSvc' -StartupType Manual -ErrorAction Continue`
` Set-Service -Name 'W32Time' -StartupType Manual -ErrorAction Continue`
` Set-Service -Name 'autotimesvc' -StartupType Manual -ErrorAction Continue`
` Set-Service -Name 'vmictimesync' -StartupType Manual -ErrorAction Continue`

Is it possible one of THESE is causing the issue, or more of them???

**UPDATE**....  So, looks like the ONLY Service I have of the above on my system is "TimeBroker" and it's currently running, it's set to "Trigger Start".
The "autotimesvc" could be the "Auto Time Zone Updater"...?  If so, I have it to, it's set to "Disabled" which I think I set in Windows Settings myself.
But I don't think I had set that on the previous install....  But, it shouldn't be effecting my System Time.

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply.

Thanks for the detailed explanation, in new versions of the file I did not go as agressive in turning off services, so it shouldn't have these issues anymore.
